hub_mozita.py

```diff
@@ -26,23 +28,45 @@ def fix_username(username):
         username = "@" + username
     return username
 
-
+def safe_conf_get(config_parser, section, key_name):
+    '''
+    returns parsed value if key_name exists in config.ini, null otherwise
+    '''
+    try:
+        return config_parser.get(section, key_name)
+    except Exception:
+        print(key_name + " non presente nella sezione " + section + "!")
+        exit()
+
+######################
+#  LOADING SECRETS   #
+######################
+# managing config.ini
 if not os.path.isfile("config.ini"):
     print(
         "Il file di configurazione non è presente.\n" +
-        "Rinomina il file 'config-sample.ini' in 'config.ini' e inserisci il token.".encode("utf-8"))
+        "Rinomina il file 'config-sample.ini' in 'config.ini' e inserisci i dati mancanti.".encode("utf-8"))
     exit()
 
+# useful object to manage secret values
 script_path = os.path.dirname(os.path.realpath(__file__))
 config_parser = ConfigParser()
 config_parser.read(os.path.join(script_path, "config.ini"))
+localtime = datetime.now()
+data_salvataggio = localtime.strftime("%Y_%m_%d")
+    
+###########################
+#  MANAGING BOT CONSTANTS #
+###########################
+TOKEN = safe_conf_get(config_parser, "bot", "TOKEN")
```

Comment:
Il file config attuale https://github.com/MozillaItalia/mozitahub_bot/blob/master/config-sample.ini ha già una sezione per il token di telegram, è un altro token o è lo stesso?

Reply:
è un altro token, va "aggiunto" quello vecchio.

In pratica:
- prendi il file nuovo
- lascia tutto così a parte TOKEN

- prendi il file vecchio
- copia il token che c'è già

- riprendi il file nuovo e incolla il token che hai copiato

Reply:
Il problema che questa linea è un duplicato di https://github.com/MozillaItalia/mozitahub_bot/blob/master/hub_mozita.py#L41
Stiamo creando un campo di opzioni per lo stesso valore che è già letto e caricato nel bot.